relayer/src/chain/cosmos.rs

```diff
@@ -1009,13 +1010,14 @@ impl ChainEndpoint for CosmosSdkChain {
 
     fn send_messages_and_wait_check_tx(
         &mut self,
-        proto_msgs: Vec<Any>,
+        tracked_msgs: TrackedMsgs,
     ) -> Result<Vec<Response>, Error> {
         crate::time!("send_messages_and_wait_check_tx");
-        debug!(
-            "send_messages_and_wait_check_tx with {} messages",
-            proto_msgs.len()
-        );
+
+        let span = span!(Level::DEBUG, "send", id = %tracked_msgs);
```

Comment:
I chose the names `send` and `send_w` for brevity, agree not the most straightforward choice.

Alternatives, feel free to choose whichever seems more appropriate:
- `send_messages_commit` (instead of `send_w`) and `send_messages_check`
- `send_commit` (instead of `send_w`) and `send_check`